packages/nextjs/components/Header.tsx

```diff
@@ -20,14 +56,65 @@ const NavLink = ({ href, children }: { href: string; children: React.ReactNode }
   );
 };
 
+const NavLinks = ({ setIsOpen }: { setIsOpen: React.Dispatch<React.SetStateAction<boolean>> }) => {
+  return (
+    <>
+      {menuItems.map((item, index) => {
+        return (
+          <li key={index}>
+            <NavLink href={item.link} setIsOpen={setIsOpen}>
+              {item.icon}
+              {item.label}
+            </NavLink>
+          </li>
+        );
+      })}
```

Comment:
maybe we can inline it in the Header itself, instead of creating a separate component for only mapping over `menuItems` ? but upto you  🙌